bson/json_util.py

```diff
@@ -80,6 +80,7 @@
 import base64
```

Comment:
Should we update the module docstring at the top to call out the canonical extended JSON format? I think a link to the spec and an example input/output would be in order.